### PR TITLE
After painfully distro hopping ...

### PR DESCRIPTION
**Pull request by sprucecellodev125**

### CODE REVIEW

File: docs/SSL/lavalink-with-ssl.md [hunk 1]
```diff
@@ -99,3 +99,20 @@ Port : 443
...
+Secure : true
+```
+
+Version 3.7.9
+```bash
+Host : penacony.hehe.rest
+Port : 443
+Password : "youshallnotpass"
+Secure : true
+```
```
Comment (DarrenOfficial) [inline docs/SSL/lavalink-with-ssl.md:118]: It's a duplicate?

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]

File: docs/SSL/lavalink-with-ssl.md (hunk 1 above)
Comment (sprucecellodev125) [reply docs/SSL/lavalink-with-ssl.md:118]: oh my bad

**Comment by sprucecellodev125** [comment]
is it okay now?

**Comment by DarrenOfficial** [comment]
```bash
* processing: https://penacony4.hehe.rest
*   Trying 104.21.27.73:443...
* Connected to penacony4.hehe.rest (104.21.27.73) port 443
* ALPN: offers h2,http/1.1
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/pki/tls/certs/ca-bundle.crt
*  CApath: none
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_256_GCM_SHA384
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=hehe.rest
*  start date: Jan  7 16:08:17 2024 GMT
*  expire date: Apr  6 16:08:16 2024 GMT
*  subjectAltName: host "penacony4.hehe.rest" matched cert's "*.hehe.rest"
*  issuer: C=US; O=Google Trust Services LLC; CN=GTS CA 1P5
*  SSL certificate verify ok.
* using HTTP/2
* h2 [:method: GET]
* h2 [:scheme: https]
* h2 [:authority: penacony4.hehe.rest]
* h2 [:path: /]
* h2 [user-agent: curl/8.2.1]
* h2 [accept: */*]
* Using Stream ID: 1
> GET / HTTP/2
> Host: penacony4.hehe.rest
> User-Agent: curl/8.2.1
> Accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
< HTTP/2 301 
< date: Mon, 26 Feb 2024 11:05:45 GMT
< content-type: text/html
< location: https://discord-linux.com
< cf-cache-status: DYNAMIC
< report-to: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v3?s=poZZoidZzb6tyLRn%2FfVe9soHZ1RJJMvvJwI1ePJg2TDhgM6fkgCD2wQWKDO0Gt4mfSnW8dafaUtaXZBY04razaYTVVdjS2u7HSfDlELQlSQNbktrKGYDI%2FdROOPVx3ZXL%2Fg2Fbu6"}],"group":"cf-nel","max_age":604800}
< nel: {"success_fraction":0,"report_to":"cf-nel","max_age":604800}
< server: cloudflare
< cf-ray: 85b7b6f7ed289f68-SIN
< alt-svc: h3=":443"; ma=86400
< 
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>openresty</center>
</body>
</html>
```

**Comment by sprucecellodev125** [comment]
> ah free arm vps (maybe), try use another hosting, ur domain https://penacony4.hehe.rest , redirected to another website.

Both of three lavalink server are hosted on my old laptop with tunnel. I'll edit this and remove the v4 one. Expect a long downtime with my lavalink server as I'll move it to my new server

**Comment by sprucecellodev125** [comment]
Omg my phone touchscreen ...

**Comment by sprucecellodev125** [comment]
okay so my lavalink server is up and running now

**Comment by AjieDev** [comment]
nice

**Review by DarrenOfficial — APPROVED** [review]

**Pull request merged**